modules/zip/src/parse-zip/zip64-info-generation.ts

```diff
@@ -0,0 +1,96 @@
+export const signature = new Uint8Array([0x01, 0x00]);
+
+/** info that can be placed into zip64 field */
+type Zip64Options = {
+  /** Original uncompressed file size and Size of compressed data */
+  size?: number;
+  /** Offset of local header record */
+  offset?: number;
+};
+
+/**
+ * creates zip64 extra field
+ * @param options info that can be placed into zip64 field
+ * @returns buffer with field
+ */
+export const createZip64Info = (options: Zip64Options): ArrayBuffer => {
```

Comment:
Nit: Prefer function syntax for longer functions. IIMHO, the  => arrow notation is best for callbacks and one liners.,
```suggestion
export function createZip64Info(options: Zip64Options): ArrayBuffer {
```